app/controllers/download_controller.rb

```diff
@@ -10,7 +10,8 @@ class DownloadController < ApplicationController
   helper_method :government_orgs_local_authorities
 
   def index
-    @custom_dimension = "#{@register.name} - download"
+    @custom_dimension2 = "#{@register.name} - download"
+    @custom_dimension3 = @register.register_phase == 'Alpha' ? 'Alpha' : 'Live'
```

Comment:
This will erroneously label `Discovery`  registers as `Live`.  If we really have to change `Beta` to `Live` can we have a mapping that incorporates all phases? Or just use the phase as-given if @cliffshepp can handle that?

Reply:
From discussion, I was under the impression that `Discovery` registers shouldn't be present on the frontend app - hence the simpler ternary operator here. Is that an incorrect assumption?

Reply:
Yes, there are a bunch of `Discovery` registers in the production CMS so I don't think we can rely on them not being present.

Reply:
Okay - I'll move the logic into its own method and use that to label the custom dimension with one of the three phases.